diff --git a/tools/rust/build_rust.py b/tools/rust/build_rust.py
index 7b40eb462a..145131e648 100755
--- a/tools/rust/build_rust.py
+++ b/tools/rust/build_rust.py
@@ -689,11 +689,6 @@ def main():
             return 1
 
     debian_sysroot = None
-    if sys.platform.startswith('linux') and not args.sync_for_gnrt:
-        # Fetch sysroot we build rustc against. This ensures a minimum supported
-        # host (not Chromium target). Since the rustc linux package is for
-        # x86_64 only, that is the sole needed sysroot.
-        debian_sysroot = DownloadDebianSysroot('amd64', args.skip_checkout)
 
     # Require zlib compression.
     if sys.platform == 'win32':
