diff --git a/tools/rust/update_rust.py b/tools/rust/update_rust.py
index b7e32f0dc7..8be7c534b5 100755
--- a/tools/rust/update_rust.py
+++ b/tools/rust/update_rust.py
@@ -75,8 +75,8 @@ def GetRustClangRevision():
 
 # Get the version of the toolchain package we already have.
 def GetStampVersion():
-    if os.path.exists(VERSION_STAMP_PATH):
-        with open(VERSION_STAMP_PATH) as version_file:
+    if os.path.exists(VERSION_SRC_PATH):
+        with open(VERSION_SRC_PATH) as version_file:
             existing_stamp = version_file.readline().rstrip()
         version_re = re.compile(r'rustc [0-9.]+ [0-9a-f]+ \((.+?) chromium\)')
         match = version_re.fullmatch(existing_stamp)
