diff --git a/tools/rust/build_rust.py b/tools/rust/build_rust.py
index f86a288b2d..d1b04cab81 100755
--- a/tools/rust/build_rust.py
+++ b/tools/rust/build_rust.py
@@ -504,7 +504,7 @@ def RustTargetTriple():
     elif sys.platform == 'win32':
         return 'x86_64-pc-windows-msvc'
     else:
-        return 'x86_64-unknown-linux-gnu'
+        return 'aarch64-unknown-linux-gnu'
 
 
 # Build the LLVM libraries and install them .
