diff --git a/tools/clang/scripts/build.py b/tools/clang/scripts/build.py
index ef2049be18..abc381295e 100755
--- a/tools/clang/scripts/build.py
+++ b/tools/clang/scripts/build.py
@@ -832,10 +832,6 @@ def main():
       '-DBUG_REPORT_URL=' + BUG_REPORT_URL,
       # See crbug.com/1126219: Use native symbolizer instead of DIA
       '-DLLVM_ENABLE_DIA_SDK=OFF',
-      # Link all binaries with lld. Effectively passes -fuse-ld=lld to the
-      # compiler driver. On Windows, cmake calls the linker directly, so there
-      # the same is achieved by passing -DCMAKE_LINKER=$lld below.
-      '-DLLVM_ENABLE_LLD=ON',
       # The default value differs per platform, force it off everywhere.
       '-DLLVM_ENABLE_PER_TARGET_RUNTIME_DIR=OFF',
       # Don't use curl.
