diff --git a/DEPS b/DEPS
index faa0c930d8..89dc4b9d20 100644
--- a/DEPS
+++ b/DEPS
@@ -747,16 +747,6 @@ deps = {
     'dep_type': 'cipd',
     'condition': 'host_os == "win"',
   },
-  'src/buildtools/reclient': {
-    'packages': [
-      {
-        'package': Var('reclient_package') + '${{platform}}',
-        'version': Var('reclient_version'),
-      }
-    ],
-    'condition': 'non_git_source',
-    'dep_type': 'cipd',
-  },
 
   # We don't know target_cpu at deps time. At least until there's a universal
   # binary of httpd-php, pull both intel and arm versions in DEPS and then pick
