diff --git a/DEPS b/DEPS
index 146d7a0fa8..c1ae9652d2 100644
--- a/DEPS
+++ b/DEPS
@@ -1018,16 +1018,6 @@ deps = {
     'dep_type': 'cipd',
     'condition': 'host_os == "win"',
   },
-  'src/buildtools/reclient': {
-    'packages': [
-      {
-        'package': Var('reclient_package') + '${{platform}}',
-        'version': Var('reclient_version'),
-      }
-    ],
-    'condition': '(host_cpu != "arm64" or host_os == "mac") and non_git_source',
-    'dep_type': 'cipd',
-  },
 
   # We don't know target_cpu at deps time. At least until there's a universal
   # binary of httpd-php, pull both intel and arm versions in DEPS and then pick
