diff --git a/chrome/browser/extensions/api/BUILD.gn b/chrome/browser/extensions/api/BUILD.gn
index 691d5f0397..6c08ca05e0 100644
--- a/chrome/browser/extensions/api/BUILD.gn
+++ b/chrome/browser/extensions/api/BUILD.gn
@@ -38,7 +38,6 @@ group("api_implementations") {
     "//chrome/browser/extensions/api/extension_action",
     "//chrome/browser/extensions/api/font_settings",
     "//chrome/browser/extensions/api/idltest",
-    "//chrome/browser/extensions/api/mdns",
     "//chrome/browser/extensions/api/module",
     "//chrome/browser/extensions/api/networking_private",
     "//chrome/browser/extensions/api/notifications",
