diff --git a/chrome/browser/extensions/api/BUILD.gn b/chrome/browser/extensions/api/BUILD.gn
index 8a96e80fbd..cae7a8bdd8 100644
--- a/chrome/browser/extensions/api/BUILD.gn
+++ b/chrome/browser/extensions/api/BUILD.gn
@@ -36,7 +36,6 @@ group("api_implementations") {
     "//chrome/browser/extensions/api/experimental_ai_data",
     "//chrome/browser/extensions/api/font_settings",
     "//chrome/browser/extensions/api/idltest",
-    "//chrome/browser/extensions/api/mdns",
     "//chrome/browser/extensions/api/module",
     "//chrome/browser/extensions/api/networking_private",
     "//chrome/browser/extensions/api/omnibox",
