diff --git a/chrome/browser/extensions/api/BUILD.gn b/chrome/browser/extensions/api/BUILD.gn
index 6173ec0a17..73adb496d6 100644
--- a/chrome/browser/extensions/api/BUILD.gn
+++ b/chrome/browser/extensions/api/BUILD.gn
@@ -34,7 +34,7 @@ group("api_implementations") {
     "//chrome/browser/extensions/api/downloads_internal",
     "//chrome/browser/extensions/api/experimental_ai_data",
     "//chrome/browser/extensions/api/font_settings",
-    "//chrome/browser/extensions/api/mdns",
+    # "//chrome/browser/extensions/api/mdns",
     "//chrome/browser/extensions/api/module",
     "//chrome/browser/extensions/api/networking_private",
     "//chrome/browser/extensions/api/omnibox",
