diff --git a/base/allocator/partition_allocator/partition_alloc.gni b/base/allocator/partition_allocator/partition_alloc.gni
index c76d412768..de4c16e75e 100644
--- a/base/allocator/partition_allocator/partition_alloc.gni
+++ b/base/allocator/partition_allocator/partition_alloc.gni
@@ -29,8 +29,7 @@ if (is_nacl) {
   assert(false, "Unknown CPU: $current_cpu")
 }
 
-has_memory_tagging =
-    current_cpu == "arm64" && is_clang && !is_asan && (is_linux || is_android)
+has_memory_tagging = false
 
 declare_args() {
   # Causes all the allocations to be routed via allocator_shim.cc. Usually,
