diff --git a/src/base/build_config.h b/src/base/build_config.h
index 6733302..db75768 100644
--- a/src/base/build_config.h
+++ b/src/base/build_config.h
@@ -35,11 +35,7 @@
 #define V8_HAS_PTHREAD_JIT_WRITE_PROTECT 0
 #endif
 
-#if defined(V8_OS_LINUX) && defined(V8_HOST_ARCH_X64)
-#define V8_HAS_PKU_JIT_WRITE_PROTECT 1
-#else
 #define V8_HAS_PKU_JIT_WRITE_PROTECT 0
-#endif
 
 #if defined(V8_TARGET_ARCH_IA32) || defined(V8_TARGET_ARCH_X64)
 #define V8_TARGET_ARCH_STORES_RETURN_ADDRESS_ON_STACK true
